Kentico.Kontent.Delivery/Builders/DeliveryOptions/IDeliveryOptionsBuilder.cs

```diff
@@ -59,7 +59,15 @@ public interface IOptionalDeliveryConfiguration : IDeliveryOptionsBuild
         /// However, the request might take longer than usual to complete.
         /// </summary>
         IOptionalDeliveryConfiguration WaitForLoadingNewContent();
-
+        
+        /// <summary>
+        /// Provide information on how many items are there in total for a query to items endpoint.
```

Comment:
Include the total number of items matching the search criteria in the response. This behavior can also be enabled for individual requests with the <see cref="IncludeTotalCountParameter"/>. Please note that using this option might increase the response time.